Bender.yml

```diff
@@ -19,6 +19,7 @@ sources:
   # Files in level 1 only depend on files in level 0, files in level 2 on files in levels 1 and 0,
   # etc. Files within a level are ordered alphabetically.
   # Level 0
+  - src/plru_tree.sv
```

Comment:
Files within one level should be sorted alphabetically to make their position well defined.

Reply:
Aha